### PR TITLE
gql 3.5.0 :snowflake:

### PR DESCRIPTION
**Pull request by psteyer**
gql 3.5.0 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-6049]
- dev_url:         https://github.com/graphql-python/gql
- conda_forge:    https://github.com/conda-forge/gql-feedstock
- pypi:           https://pypi.org/project/gql/3.5.0
- pypi inspector: https://inspector.pypi.io/project/gql/3.5.0

### Explanation of changes:

- feedstock added to aggregate
- moving entry point

[PKG-6049]: https://anaconda.atlassian.net/browse/PKG-6049?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -163,18 +198,20 @@ outputs:
...
       run:
         - {{ pin_subpackage('gql-with-aiohttp', exact=True) }}
+        - {{ pin_subpackage('gql-with-httpx', exact=True) }}
         - {{ pin_subpackage('gql-with-botocore', exact=True) }}
         - {{ pin_subpackage('gql-with-requests', exact=True) }}
         - {{ pin_subpackage('gql-with-websockets', exact=True) }}
         - {{ pin_subpackage('gql', exact=True) }}
-        - {{ min_python }}
+        - python
     test:
```
Comment (ViridianMelody) [inline recipe/meta.yaml:206]:
```suggestion
    test:
      imports:
        - gql
        - gql.transport.httpx
        - gql.transport.aiohttp
        - gql.transport.requests
        - gql.transport.appsync_auth
        - gql.transport.websockets
```

This should satisfy the linter, and a bit more testing never hurts.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,27 +1,26 @@
...
-    - {{ min_python }}
+    - python
     - pip
+    - wheel
+    - setuptools
   run:
-    - {{ min_python }}
+    - python
 
 test:
```
Comment (ViridianMelody) [inline recipe/meta.yaml]: Best to remove the mock test in the global output or replace it with a comment. It's not being run anyway.

**Review by ViridianMelody — APPROVED** [review]
Looks good to me, just couple of small suggestions that'll satisfy the linter.

**Review by Arishamays1 — APPROVED** [review]

**Review by ViridianMelody — APPROVED** [review]

File: recipe/meta.yaml [hunk 3]
```diff
@@ -188,22 +228,32 @@ outputs:
...
         - gql-cli --help
         - gql-cli --version
-        - pytest tests -vv --no-cov-on-fail --cov gql --cov-report=term-missing:skip-covered --cov-fail-under=99
-
+        - gql
+        - gql.transport.httpx
+        - gql.transport.aiohttp
+        - gql.transport.requests
+        - gql.transport.appsync_auth
+        - gql.transport.websockets
```
Comment (ViridianMelody) [inline recipe/meta.yaml]:
```suggestion
```
Oop, these got copied into both places from my faulty suggestion comment.

**Review by ViridianMelody — APPROVED** [review]

**Pull request merged**